src/pages/task/logs/LogTypes.tsx

```diff
@@ -206,64 +224,34 @@ const useRenderBody: React.FC<{
             )}
           </ButtonContainer>
         ) : null}
-        <Radio
-          id="cy-task-radio"
-          value={LogTypes.Task}
-          onClick={() =>
-            taskAnalytics.sendEvent({
-              name: "Select Logs Type",
-              logsType: LogTypes.Task,
-            })
-          }
-        >
+        <Radio id="cy-task-radio" value={LogTypes.Task}>
           Task Logs
         </Radio>
-        <Radio
-          id="cy-agent-radio"
-          value={LogTypes.Agent}
-          onClick={() =>
-            taskAnalytics.sendEvent({
-              name: "Select Logs Type",
-              logsType: LogTypes.Agent,
-            })
-          }
-        >
+        <Radio id="cy-agent-radio" value={LogTypes.Agent}>
           Agent Logs
         </Radio>
-        <Radio
-          id="cy-system-radio"
-          value={LogTypes.System}
-          onClick={() =>
-            taskAnalytics.sendEvent({
-              name: "Select Logs Type",
-              logsType: LogTypes.System,
-            })
-          }
-        >
+        <Radio id="cy-system-radio" value={LogTypes.System}>
           System Logs
         </Radio>
-        <Radio
-          id="cy-event-radio"
-          value={LogTypes.Event}
-          onClick={() =>
-            taskAnalytics.sendEvent({
-              name: "Select Logs Type",
-              logsType: LogTypes.Event,
-            })
-          }
-        >
+        <Radio id="cy-event-radio" value={LogTypes.Event}>
           Event Logs
         </Radio>
       </StyledRadioGroup>
       {hideLogs ? (
-        <div id="cy-no-logs">No logs found</div>
+        <div data-cy="cy-no-logs">No logs found</div>
       ) : (
         <LogContainer>
-          {data.map((d) =>
+          {data.map((d, index) =>
             d.kind === "taskEventLogEntry" ? (
-              <TaskEventLogLine key={uuid()} {...d} />
+              <TaskEventLogLine
+                key={`${d.resourceId}_${d.id}_${index}`} // eslint-disable-line react/no-array-index-key
```

Comment:
it may be okay to omit `${d.resourceId}_${d.id}_` and rely on index only since we don't expect list elements to change order